kbase-extension/scss/partials/_narrative.scss

```diff
@@ -70,7 +51,6 @@
 }
 
 .dz-file__progress__bar {
-  width: 0;
```

Comment:
@eamahanna this is the fix for the bug that you spotted with the progress bar not growing